include/zephyr/net/wifi_mgmt.h

```diff
@@ -83,6 +83,8 @@ enum net_request_wifi_cmd {
 	NET_REQUEST_WIFI_CMD_CHANNEL,
```

Comment:
There should be no need for the mac setter/getter for wifi. As the wifi interface is built on top of ethernet interface, there is already MAC address setter function in `include/zephyr/net/ethernet_mgmt.h`
And the MAC address can be fetched by `net_if_get_link_addr()` function.
The net-shell has function to set the MAC address, but if you think it is missing some functionality, please propose a separate pr. The net-shell file is `subsys/net/lib/shell/iface.c`

Reply:
We can use functionality from net-shell, Thanks for pointing out.